server/src/main/java/io/druid/segment/realtime/plumber/RealtimePlumber.java

```diff
@@ -352,7 +380,11 @@ public void doRun()
           {
             try {
               for (Pair<FireHydrant, Interval> pair : indexesToPersist) {
-                metrics.incrementRowOutputCount(persistHydrant(pair.lhs, schema, pair.rhs));
+                metrics.incrementRowOutputCount(
```

Comment:
Will we lose data if one hydrant is persisted with the metadata, then the plumber crashes? If I'm reading the code right, that would cause the next bootstrap to think that all the previously read data was persisted.